### PR TITLE
docs(components): [select] add width change description

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

The issue with width changes seems to affect a lot of people, and it might be friendlier to add a note at the top of the page.

### CODE REVIEW

File: docs/en-US/component/select.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ When there are plenty of options, use a drop-down menu to display and select des
 
 :::tip
 
+After version 2.5.0, due to the refactoring of `el-select`, the default width becomes `100%`. When used in a form, due to layout problems, the width of `el-select` will become `11px`. In order to keep the layout consistent with the previous version, perhaps You need to give `el-select` a width value.
```
Comment (kooriookami) [inline docs/en-US/component/select.md:12]:
```suggestion
After version `2.5.0`, the default width of `el-select` becomes `100%`. When used in a inline form, the width will collapse. In order to display the width properly, you need to give `el-select` a specific width.
```

**Comment by kooriookami** [comment]
Do we need two tips?
image

**Comment by btea** [comment]
Needed, these are instructions for two different scenarios.

**Comment by kooriookami** [comment]
OK~

**Review by kooriookami — APPROVED** [review]

**Pull request merged**